lib/app.js

```diff
@@ -147,7 +161,10 @@ module.exports = class App {
         return fs.mkdirpAsync(path.dirname(test.referencePath))
             .then(() => fs.copyAsync(test.currentPath, test.referencePath))
             .then(() => {
-                this._runner.emit('updateResult', result);
+                if (this._runner) {
+                    // if applies reused result and haven't run any tests before this then no runner present
```

Comment:
не совсем понятный комментрий и он по моему мнению долежн быть на строку выше и объяснять для чего нужна такая проверка. Я бы так написал:
```js
// the runner may be absent in case when result is reused and the tests haven't run yet
this._runner && this._runner.emit('updateResult', result);
```

Reply:
Перепишу коммент ещё раз